prereise/gather/winddata/hrrr/helpers.py

```diff
@@ -0,0 +1,37 @@
+def formatted_filename(dt, product, hours_forecasted):
+    """Deterministically returns a filename
+
+    :param datetime.datetime dt:
+    :param string product:
+    :param string hours_forecasted:
+
+    :return: (*str*)
+    """
+    return f"{dt.strftime('%Y_%m_%d_%Hhr')}_{product}_{hours_forecasted}.grib2"
+
+
+def generate_index_url(url):
+    """Adds .idx to end of a url
+
+    :param string url:
+
+    :return: (*str*)
+    """
+    return f"{url}.idx"
```

Comment:
Do we really need a function explicitly doing this?

Reply:
I usually base this on how many usages there are. In this case I only see one usage, so agree it doesn't need to be a function.